backend_new/prisma/schema.prisma

```diff
@@ -0,0 +1,1025 @@
+  paperApplications             PaperApplications[]
+
+  @@map("application_methods")
+}
+
+// Note: [additional_phone_number, additional_phone_number_type, household_size] formerly max length 16; 
+//       [contact_preferences, income] formerly max length 64;
+//       [app_url] formerly max length 256; 
+// Note: [contact_preferences] formerly max array length of 8
+// Note: [household_member] formerly had max array lenght of 32
+// Note: missing virtual [flagged] field
+model Applications {
+  id                           String                        @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt                    DateTime                      @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt                    DateTime                      @updatedAt @map("updated_at") @db.Timestamp(6)
+  deletedAt                    DateTime?                     @map("deleted_at") @db.Timestamp(6)
+  appUrl                       String?                       @map("app_url")
+  additionalPhone              Boolean?                      @map("additional_phone")
+  additionalPhoneNumber        String?                       @map("additional_phone_number")
+  additionalPhoneNumberType    String?                       @map("additional_phone_number_type")
+  contactPreferences           String[]                      @map("contact_preferences")
+  householdSize                Int?                          @map("household_size")
+  housingStatus                String?                       @map("housing_status")
+  sendMailToMailingAddress     Boolean?                      @map("send_mail_to_mailing_address")
+  householdExpectingChanges    Boolean?                      @map("household_expecting_changes")
+  householdStudent             Boolean?                      @map("household_student")
+  incomeVouchers               Boolean?                      @map("income_vouchers")
+  income                       String?
+  incomePeriod                 IncomePeriodEnum              @map("income_period")
+  preferences                  Json
+  programs                     Json?
+  status                       ApplicationStatusEnum
+  language                     LanguagesEnum
+  submissionType               ApplicationSubmissionTypeEnum @map("submission_type")
+  acceptedTerms                Boolean?                      @map("accepted_terms")
+  submissionDate               DateTime?                     @map("submission_date") @db.Timestamptz(6)
+  // if this field is true then the application is a confirmed duplicate
+  // meaning that the record in the application flagged set table has a status of duplicate
+  markedAsDuplicate            Boolean                       @default(false) @map("marked_as_duplicate")
+  confirmationCode             String                        @map("confirmation_code")
+  reviewStatus                 ApplicationReviewStatusEnum   @default(valid) @map("review_status")
```

Comment:
Was optional previously

Reply:
`@Column({ enum: ApplicationReviewStatus, default: ApplicationReviewStatus.valid })` it was optional at the DTO level but on the db level it was required with a default in case of null